docs/rst/user_manual/configuration.rst

```diff
@@ -685,6 +685,93 @@ It will use such profile for configuring the Participant.
 
 .. _user_manual_configuration_general_example:
 
+
+Forwarding Routes
+=================
+
+The |ddsrouter| is capable of establishing different internal routes between its participants.
+This feature enables users to only send sensible data to a set of participants.
```

Comment:
```suggestion
This feature enables users to only forward sensible data to a set of participants.
```

Reply:
I would use "forward" instead everywhere, to avoid confusion with writers' send method.